assets/src/components/form/layouts/two-column-admin/form-section.js

```diff
@@ -51,20 +52,37 @@ const FormSection = ( {
 	const classes = htmlClass ?
 		`${ htmlClass } ee-form-section px-0 pt-3 border rounded` :
 		'ee-form-section px-0 pt-3 border rounded';
-	const formInfo = useMemo( () => showRequiredNotice && (
-		<FormInfo
-			key="formInfo"
-			formInfo={ requiredNotice }
-			dismissable={ false }
-		/>
-	), [ showRequiredNotice ] );
-	const formErrors = hasValidationErrors ? (
-		<FormInfo
-			formInfo={ errorsNotice }
-			dashicon={ 'warning' }
-			dismissable={ false }
-		/>
-	) : null;
+	const formInfo = useMemo(
+		() => formInfoNotice ? (
```

Comment:
why not `formInfoNotice && ( ...` or `!! formInfoNotice && ( ...`? to be safer?

Reply:
I may have done something wrong but at some point in other components I had been doing something like:

```js
const thing = showThing && <Thing />;

return (
	<SomeComponent>
		{ thing }
	</SomeComponent>
);
```

and when `showThing` was `false` I would get `0` rendered where `{ thing }` was.

Again, it's totally possible I was doing something wrong, because that was earlier on in my budding React career, but that's why I don't use that shortcut.

I did just notice that I passed the wrong dependencies to `useMemo` so I'll be pushing changes to this in a minute

Reply:
`false` would not render anything. Just to make sure that you are making a boolean comparison, add `!!` before the expression.

Reply:
Yeah, or sometimes I do this:
```
const thing = Boolean(showThing) && <Thing />;
```
If I know that `showThing` might have value `0`.
In this example `0` will be coerced to `false`.

Reply:
right I was probably getting a non-boolean falsey value...

I'll update this to use `!! formInfoNotice && `